### ISSUE TITLE
Config options not being updated

### ISSUE THREAD

**Issue by Kevin125566** [CHR15cs/CS2-Practice-Plugin#53]
Changing the CSPracc.json file doesn't change anything. I set AdminRequirements to false and when I try to test the commands on the server, I still receive "Only admins can execute this command"

**Comment by Kevin125566** [comment]
Yes, but I think I figured out what's wrong. When I leave the version number to 1, the admin requirements config doesn't work. It only works when I change it to a different number

**Comment by CHR15cs** [comment]
Thats true, I check the ConfigVersion in case someuses a legacy config file which do not include the flag, might adjust the handling tho
AdminRequirement need ConfigVersion to be > 1
